site/ja/lite/microcontrollers/get_started.md

```diff
@@ -0,0 +1,329 @@
+アプリケーション例を始めから終わりまで見ていきます。
+
+## Hello World サンプル
+
+この例はマイクロコントローラ向けTensorFlow Liteの使い方の基本を実演します。
+モデルの訓練、TensorFlow Liteと共に使うためのモデル変換、
+そしてマイクロコントローラ上での推論実行までの始めから終わりまでのワークフローを含んでいます。
+
+この例では、モデルは正弦波関数を再現するように訓練されています。
+１つの数を入力として、[正弦波](https://en.wikipedia.org/wiki/Sine)の数値を出力します。
+マイクロコントローラにデプロイされると、その予測は、LEDを点滅させたりアニメーションを制御したりすることに使用されます。
+
+この例は下記を含みます。
+
+* モデルの訓練と変換の仕方を実演するJupyter notebook
+* モデルを使って推論を実行するC++ 11 アプリケーション。そのアプリケーションは、Arduino、SparkFun Edge、STM32F746G discovery kit、そしてmacOSで動作検証されています
+* 推論処理を実演する単体テスト
+
+### サンプルを動かす
+
+各デバイスのサンプルを実行するために、`README.md`の指示を見ていきます。
+<a class="button button-primary" href="https://github.com/tensorflow/tensorflow/tree/master/tensorflow/lite/micro/examples/hello_world/README.md">Hello World README.md</a>
+
+## 推論を実行する方法
+
+以下の節は *Hello World* サンプルの
+[`hello_world_test.cc`](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/micro/examples/hello_world/hello_world_test.cc)を見ていきます。
+これは、マイクロコントローラ向けTensorFlow Liteを使って推論を実行する方法を実演します。
+
+検証はモデルを読み込み、それを使い何回か推論を実行します。
+
+### ライブラリをインクルードする
+
+マイクロコントローラ向けTensorFlow Liteライブラリを使うために、以下のヘッダーファイルをインクルードする必要があります。
+
+```C++
+#include "tensorflow/lite/micro/kernels/all_ops_resolver.h"
+#include "tensorflow/lite/micro/micro_error_reporter.h"
+#include "tensorflow/lite/micro/micro_interpreter.h"
+#include "tensorflow/lite/schema/schema_generated.h"
+#include "tensorflow/lite/version.h"
+```
+
+* [`all_ops_resolver.h`](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/micro/kernels/all_ops_resolver.h)
+    モデルを実行するインタープリタで使用される演算を提供します。
+* [`micro_error_reporter.h`](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/micro/micro_error_reporter.h)
+    デバッグ情報を出力します。
+* [`micro_interpreter.h`](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/micro/micro_interpreter.h)
+    モデルを読み込んで実行するコードを含みます。
+* [`schema_generated.h`](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/schema/schema_generated.h)
+    TensorFlow Lite [`FlatBuffer`](https://google.github.io/flatbuffers/) モデルファイルフォーマットのスキームを含みます。
+* [`version.h`](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/version.h)
+    TensorFlow Lite スキームへのバージョン情報を提供します。
+
+### モデルをインクルードする
+
+マイクロコントローラ向けTensorFlow Lite インタープリタは、モデルがC++配列で提供されることを期待しています。*Hellow World* サンプルでは、モデルは `sine_model_data.h` と `sine_model_data.cc` で定義されています。ヘッダーは以下の行に含まれます。
+
+```C++
+#include "tensorflow/lite/micro/examples/hello_world/sine_model_data.h"
+```
+
+### 単体テストを用意する
+
+見ていくコードは単体テストで、それはマイクロコントローラ向けTensorFlow Liteフレームワークの単体テストフレームワークを使います。
+フレームワークを読み込むため、以下のファイルをインクルードします。
+
+```C++
+#include "tensorflow/lite/micro/testing/micro_test.h"
+```
+
+テストは以下のマクロを使って定義されます。
+
+```C++
+TF_LITE_MICRO_TESTS_BEGIN
+
+TF_LITE_MICRO_TEST(LoadModelAndPerformInference) {
+```
+
+コードの残り部分は、モデルの読み込みと推論を実演します。
+
+### ログ取得を用意する
+
+ログ取得の準備するために、`tflite::ErrorReporter` ポインタが作成され、ポインタは `tflite::MicroErrorReporter` インスタンスに対して使用されます。
```

Comment:
[suggeston]
```suggestion
ログ取得の準備をするために、`tflite::MicroErrorReporter` インスタンスのポインタを用いて `tflite::ErrorReporter` ポインタが作成されます。
```

I think `tflite::ErrorReporter` is an abstract class and `tflite::MicroErrorReporter` is one of implementations of it.

Reply:
I used your suggestion.

Reply:
I used this(https://github.com/tensorflow/docs-l10n/pull/123#discussion_r392656930) sugession .